source/code-snippets/aggregation/agg.js

```diff
@@ -0,0 +1,48 @@
+const { MongoClient } = require('mongodb');
+const uri = process.env.DRIVER_REF_URI;
```

Comment:
Is `DRIVER_REF_URI` used in other examples? Normally I've seen this as `MONGODB_URI`. Nothing wrong here just thought it might be an inconsistency.

Reply:
I think `DRIVER_REF_URI` was taken from some Java docs but I will update it to `MONGODB_URI`.